style/install-linters.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# A script to install static analysis tools pylint, shellcheck, and cppcheck.
+
+# Install cppcheck-1.82-1
+sudo apt-get install cppcheck=1.82-1 -y
+cppcheck --version
+
+# Install shellcheck-v0.7.0
+sudo apt install xz-utils
+wget -qO- "https://github.com/koalaman/shellcheck/releases/download/v0.7.0/shellcheck-v0.7.0.linux.x86_64.tar.xz" | tar -xJv
+sudo cp "shellcheck-v0.7.0/shellcheck" /usr/bin/
+rm -r -f shellcheck-v0.7.0
+shellcheck --version
+
+# Install pylint 2.4.4
+sudo apt install python3-pip
+pip3 --version
+pip3 install pylint==2.4.4
+export PATH=$HOME/.local/bin:$PATH
+pylint --version
```

Comment:
```suggestion
pylint --version
echo export PATH="$HOME"/.local/bin:"$PATH" >> ~/.bashrc
echo "Please run source ~/.bashrc"
```
This is because exporting in a bash script does not export in the actual shell. We need the user to export it so they can run `pylint` on their own. We do this in the `install.sh` of onvm.